virttest/qemu_devices/qcontainer.py

```diff
@@ -1005,7 +1005,9 @@ def machine_q35(cmd=False):
             logging.warn('Using Q35 machine which is not yet fully tested on '
                          'avocado-vt. False errors might occur.')
             devices = []
-            bus = (qdevices.QPCIEBus('pcie.0', 'PCIE', root_port_type, 'pci.0'),
+            pcie_root_port_params = params.get('pcie_root_port_params')
+            bus = (qdevices.QPCIEBus('pcie.0', 'PCIE', root_port_type,
+                                     pcie_root_port_params, 'pci.0'),
```

Comment:
A good catch~  I change the order of new params to avoid effect other call.
However, i did not apply it with aarch64, since no requirement by now. thanks!

Reply:
I thought of this issue again,  and apply it for aarch64 now.  No effect for arm64_pci unless user define the value, so i think it is ok. How do you think ?

Reply:
I saw you changed codes 
```
    def __init__(self, busid, bus_type, root_port_type,
                 aobject=None, root_port_params=None):
```
it looks good to me, for add new interface arguments, the better way is to append then default it at the end since this will avoid mismatch error that the called place was used by positional argument.

Reply:
> ,

Hi, @PaulYuuu 
maybe you know of aarch64 then me, does the aarch64 need to this requirement?
hope @yanan-fu could support it if @PaulYuuu confirms that aarch64 needs.

Thanks. cc @zhenyzha

Reply:
Yes, aarch64 should also be in the support list.
```
# /usr/libexec/qemu-kvm -M virt -device pcie-root-port,?
pcie-root-port options:
  addr=<int32>           - Slot and optional function number, example: 06.0 or 06 (default: -1)
  aer_log_max=<uint16>   -  (default: 8)
  bus-reserve=<uint32>   -  (default: 4294967295)
  chassis=<uint8>        -  (default: 0)
  disable-acs=<bool>     -  (default: false)
  failover_pair_id=<str>
  hotplug=<bool>         -  (default: true)
  io-reserve=<size>      -  (default: 18446744073709551615)
  mem-reserve=<size>     -  (default: 18446744073709551615)
  multifunction=<bool>   - on/off (default: false)
  port=<uint8>           -  (default: 0)
  power_controller_present=<bool> - on/off (default: true)
  pref32-reserve=<size>  -  (default: 18446744073709551615)
  pref64-reserve=<size>  -  (default: 18446744073709551615)
  rombar=<uint32>        -  (default: 1)
  romfile=<str>
  slot=<uint16>          -  (default: 0)
  x-migrate-msix=<bool>  -  (default: true)
  x-pcie-extcap-init=<bool> - on/off (default: true)
  x-pcie-lnksta-dllla=<bool> - on/off (default: true)
  x-speed=<PCIELinkSpeed> - 2_5/5/8/16 (default: "16")
  x-width=<PCIELinkWidth> - 1/2/4/8/12/16/32 (default: "32")
```